app/views/unattended/provisioning_templates/iPXE/ipxe_global_default.erb

```diff
@@ -3,6 +3,8 @@ kind: iPXE
 model: ProvisioningTemplate
 name: iPXE global default
 snippet: false
+description: |
+  TODO
```

Comment:
@lzap could you please provide a suggestion?

Reply:
Just copy PXELinux global boot description and replace with iPXE. The very same purpose.